### DISCUSSION TITLE
Provider reference at header level of In-Network file

### DISCUSSION THREAD

**Discussion by SCShuler** [CMSgov/price-transparency-guide#371] [Q&A] [answered]
Can the provider-reference at the header level of the In-Network MRF be used in lieu of the provider-reference field within the Negotiated Rates Detail object? It states within the Negotiated rates detail object that either provider-group or provider-reference is required but this defeats the purpose of saving space by not having to repeat multiple provider data per rate.

**Comment by shaselton-usds** [comment, marked as answer]
@SCShuler -- moved the "issue" to a "discussion".

The `provider_references` in the [Negotiated Rate Details Object](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#negotiated-rate-details-object) is an array of IDs  referencing the `provider_group_id` in the [Provider Reference Object](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#provider-reference-object) that is found in the "header level" of the in-network MRF. 

This is to say, I think the way it is set up is the way that you're asking for it to be set up.

An example would be the [definition of a provider network](https://github.com/CMSgov/price-transparency-guide/blob/master/examples/in-network-rates/in-network-rates-fee-for-service-single-plan-sample.json#L25-L28) at the "header level" (aka, the root node) that is used later in the file [for a negotiated item/service](https://github.com/CMSgov/price-transparency-guide/blob/master/examples/in-network-rates/in-network-rates-fee-for-service-single-plan-sample.json#L87). Please note that in the example, the ID being referenced is "1", but it could have easily been "2" that points back to an externally referenced provider network.

**Comment by SCShuler** [comment]
Thanks Scott! This is very helpful.

From: scott haselton ***@***.***>
Sent: Wednesday, February 2, 2022 3:34 PM
To: CMSgov/price-transparency-guide ***@***.***>
Cc: TOM SHULER ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [CMSgov/price-transparency-guide] Provider reference at header level of In-Network file (Discussion #371)



WARNING: THIS IS AN EXTERNAL EMAIL THAT ORIGINATED OUTSIDE OF OUR EMAIL SYSTEM.
DO NOT CLICK links/attachments unless you know that the content is safe!
For suspicious emails, report using the Phish Alert Report button. For marketing/SPAM emails, delete.





@SCShuler<https://github.com/SCShuler> -- moved the "issue" to a "discussion".

The provider_references in the Negotiated Rate Details Object<https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#negotiated-rate-details-object> is an array of IDs referencing the provider_group_id in the Provider Reference Object<https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#provider-reference-object> that is found in the "header level" of the in-network MRF.

This is to say, that I think the way it is set up is the way that you're asking for it to be set up.

An example would be the definition of a provider network<https://github.com/CMSgov/price-transparency-guide/blob/master/examples/in-network-rates/in-network-rates-fee-for-service-single-plan-sample.json#L25-L28> at the "header level" (aka, the root node) that is used later in the file for a negotiated item/service<https://github.com/CMSgov/price-transparency-guide/blob/master/examples/in-network-rates/in-network-rates-fee-for-service-single-plan-sample.json#L87>. Please note that in the example, the ID being referenced is "1", but it could have easily been "2" that points back to an externally referenced provider network.

—
Reply to this email directly, view it on GitHub<https://github.com/CMSgov/price-transparency-guide/discussions/371#discussioncomment-2099627>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXGLJCQP3XC4CC5RN7W5P2DUZGINJANCNFSM5NNCIC4A>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
